.github/workflows/build.yaml

```diff
@@ -96,3 +96,10 @@ jobs:
           name: test-report
           path: build/reports/tests/test/
           if-no-files-found: ignore
+
+      - name: Upload distribution
+        uses: actions/upload-artifact@v3
+        with:
+          name: st
+          path: ./build/distributions/st.zip
```

Comment:
This might double zip :(

Reply:
Yep: https://github.com/actions/upload-artifact#zipped-artifact-downloads

Do the `installDist` output dir so it gets zipped once

Reply:
Hmm, interesting. My computer must have double-extracted then because it was invisible to me. Homebrew also seems to double-extract.